base/namedtuple.jl

```diff
@@ -89,6 +89,7 @@ firstindex(t::NamedTuple) = 1
 lastindex(t::NamedTuple) = nfields(t)
 getindex(t::NamedTuple, i::Int) = getfield(t, i)
 getindex(t::NamedTuple, i::Symbol) = getfield(t, i)
+getindex(t::NamedTuple, r::AbstractUnitRange{T}) where T = ([getfield(t, ri) for ri in r]...,)
```

Comment:
How about
```
getfield.(t, r)
```
? Are there tests for the `getindex` methods defined above, to which you could add tests for the range index method?

Reply:
I didn't find any such test for `getindex`. Should I go forward and add them

Reply:
Okay, I'm getting `ERROR: ArgumentError: broadcasting over dictionaries and `NamedTuple`s is reserved`. So, I guess the present implementation is fine.